### PR TITLE
LGA-1342 property prefixes

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
## What does this pull request do?

Adds a property prefix so screen readers know that they are dealing with property number 1
- Sets a variable if property count > 1.
- Feeds variable into fieldset for all relevant fields.
- Radio buttons only, the currency fields pick up the fieldset legend.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: cla_public/templates/macros/form.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
     - attrs <object> (default: None)
         Custom HTML attributes for fieldset element
 #}
-{% macro fieldset(field=None, class_='govuk-fieldset ', legend=None, field_attrs={}) %}
+{% macro fieldset(field=None, class_='govuk-fieldset ', legend=None, field_attrs={}, hidden_legend_prefix=None) %}
```
Comment (mayowaadeniyi) [inline cla_public/templates/macros/form.html:16]: `hidden_legend_prefix=None`

Why is this necessary? Wouldn't the same thing happen if we left it out?

Comment (MalcolmVonMoJ) [reply cla_public/templates/macros/form.html:16]: No, this is needed.  It is a new argument in the macro that we need.  We feed in a value there on the property pages telling it what the prefix needs to be.  If nothing is received, it defaults to none.

**Review by mayowaadeniyi — APPROVED** [review]
LGTM 👍
